src/Compilers/CSharp/Portable/Parser/DocumentationCommentParser.cs

```diff
@@ -1129,7 +1129,8 @@ SyntaxToken tryEatCheckedOrHandleUnchecked(ref SyntaxToken operatorKeyword)
                 if (CurrentToken.Kind == SyntaxKind.UncheckedKeyword)
                 {
                     // if we encounter `operator unchecked`, we place the `unchecked` as skipped trivia on `operator`
-                    var misplacedToken = this.AddError(this.EatToken(), ErrorCode.ERR_MisplacedUnchecked);
+                    var code = ErrorCode.ERR_MisplacedUnchecked;
+                    var misplacedToken = this.AddError(this.EatToken(), ErrorCode.WRN_ErrorOverride, new SyntaxDiagnosticInfo(code), (int)code);
```

Comment:
There's a helper for this - `AddErrorAsWarning`

Reply:
Thanks